plugins/inputs/mesos/mesos.go

```diff
@@ -452,16 +459,18 @@ func (m *Mesos) gatherSlaveTaskMetrics(address string, defaultPort string, acc t
 	}
 
 	for _, task := range metrics {
-		tags["task_id"] = task["executor_id"].(string)
+		tags["task_id"] = task.ExecutorID
+		tags["framework_id"] = task.FrameworkID
 
 		jf := jsonparser.JSONFlattener{}
-		err = jf.FlattenJSON("", task)
+		err = jf.FlattenJSON("", task.Statistics)
 
 		if err != nil {
 			return err
 		}
+		timestamp := time.Unix(int64(jf.Fields["timestamp"].(float64)),0)
 
-		acc.AddFields("mesos-tasks", jf.Fields, tags)
+		acc.AddFields("mesos-tasks", jf.Fields, tags, timestamp)
```

Comment:
I should have caught this earlier, but this should be `mesos_tasks`, not `mesos-tasks`, could we change that? how long has it been like this for?